x-pack/plugins/viz_editor/public/pseudo_plugins/xy_chart_plugin/xy_chart_plugin.tsx

```diff
@@ -10,19 +10,19 @@ import {
   IconType,
 } from '@elastic/eui';
 import React from 'react';
-import { DatasourceField, fieldToOperation } from '../../../common';
-import { columnSummary } from '../../common/components/config_panel';
+import { DatasourceField } from '../../../common';
 import {
   Axis,
   getColumnIdByIndex,
-  selectColumn,
   UnknownVisModel,
   updatePrivateState,
   VisModel,
 } from '../../common/lib';
 import { operationToName } from '../../common/lib';
 import { getOperationsForField } from '../../common/lib/field_config';
 import { EditorPlugin, PanelComponentProps, Suggestion } from '../../editor_plugin_registry';
+import { XAxisEditor } from './xaxis_editor';
+import { YAxisEditor } from './yaxis_editor';
```

Comment:
Why are these different?

Reply:
Because they have different behaviors. Let's keep them separate for a while, and if we see that there is significant (not accidental) overlap, we can try to come up with a unified implementation.

Reply:
Discussed earlier and it makes sense.